packages/eslint-plugin/docs/rules/init-declarations.md

```diff
@@ -0,0 +1,140 @@
+# require or disallow initialization in variable declarations (`init-declarations`)
+
+In Typescript, variables can be assigned during declaration, or at any point afterwards using an assignment statement. For example, in the following code, `foo` is initialized during declaration, while `bar` is initialized later.
```

Comment:
A good example of how to write rule docs for an extension rule is here
https://github.com/typescript-eslint/typescript-eslint/blob/7d963fd846935acd91b7b0cd31c56a70a2b994d1/packages/eslint-plugin/docs/rules/no-dupe-class-members.md

Essentially - we don't want to copy+paste the docs, as they can get out of sync with ESLint's docs.
So instead we want to just mention what changes we make (like "This rule extends the base `eslint/init-declarations` rule. It adds support for TypeScript's variable declaration syntax.")

Reply:
cool, I will add a reference

Reply:
done